examples/image_classification/main.js

```diff
@@ -5,6 +5,8 @@ $(document).ready(() => {
 });
 
 $(window).load(() => {
-  // Execute inference
-  example.main();
+  if (example._currentFramework === 'WebNN') {
```

Comment:
why is it only for `WebNN`?

Reply:
For OpenCV.js, the main method was invoked by cv.onRuntimeInitialized which is the callback triggered when the OpenCV.js library is ready for use.

Reply:
got it. probably a comment here will be more helpful.

Reply:
OK, I will update by adding some comment, thanks.